demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>It might be nice to add a little more content here so it's a little clearer and the user has some "next steps" if this one wasn't quite right:</p><p> </p><ul><li>it could have a Conclusion heading</li><li>some links to other relevant demos, e.g. HinSAGE for heterogeneity, the whole link prediction folder (potentially these links could go in the opening section too, so a user can jump to another more relevant demo if they decide this one isn't quite right). I tried to do something like this at the start and end of <a href="https://stellargraph.readthedocs.io/en/latest/demos/node-classification/gcn-node-classification.html" target="_blank">https://stellargraph.readthedocs.io/en/latest/demos/node-classification/gcn-node-classification.html</a>, if you're looking for more explanation of what I mean.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
I've added something similar to the gcn node classification demo at the top and bottom